core/contracts/common/implementation/MultiRole.sol

```diff
@@ -137,6 +138,7 @@ abstract contract MultiRole {
      * managing role for `roleId`.
      */
     function addMember(uint roleId, address newMember) public onlyShared(roleId) onlyRoleManager(roleId) {
+        require(newMember != address(0x0));
```

Comment:
This `addMember` function within the `Role` lib was not specifically pointed out during the audit and this function follows a different pattern: it’s public and has different privileges on who can call it. I added the same require to keep things consistent. let me know if you have any issues with this.

Reply:
This just calls the other `addMember` function on the next line, so I think this has the effect of doing this check *twice*. I would suggest removing this one.